core/include/drivers/rtc.h

```diff
@@ -12,14 +12,29 @@
 /* The RTC allows to set/get offset for correction */
 #define RTC_CORRECTION_FEATURE	BIT(0)
 
+struct rtc_subs;
```

Comment:
I don't understand this line and didn't find any usage of this. Seems to be an oversight.